apps/services/user-notification/src/app/modules/notifications/notifications.service.ts

```diff
@@ -151,4 +163,135 @@ export class NotificationsService {
     }
     return template
   }
+
+  async findOne(
+    user: User,
+    id: number,
+    locale: string,
+  ): Promise<RenderedNotificationDto> {
+    // Retrieve the notification
+    const notification = await this.notificationModel.findOne({
+      where: {
+        id: id,
+        recipient: user.nationalId,
+      },
+    })
+
+    if (!notification) {
+      throw new NotFoundException(
```

Comment:
We created `NoContentException` to throw instead. To respond rather with `204` as described in our [API Design Guide](https://docs.devland.is/technical-overview/api-design-guide/rest-response#get)